lib/modules/manager/cargo/range.ts

```diff
@@ -0,0 +1,9 @@
+import type { RangeStrategy } from '../../../types';
+import type { RangeConfig } from '../types';
+
+export function getRangeStrategy(config: RangeConfig): RangeStrategy {
+  if (config.rangeStrategy !== 'auto') {
+    return config.rangeStrategy;
+  }
+  return 'bump';
```

Comment:
```suggestion
  return rangeStrategy === 'auto' ? 'bump' : rangeStrategy;
```

shorten, like on circleci?